modules/nw-multus-create-network-apply.adoc

```diff
@@ -26,13 +27,15 @@ spec:
     {
       "cniVersion": "0.3.1",
       "name": "work-network",
+      "namespace": "namespace2", #<1>
       "type": "host-device",
       "device": "eth1",
       "ipam": {
         "type": "dhcp"
       }
     }
 ----
+<1> Optional: You can specify a namespace to which the NAD is applied. If you are working in the namespace where the NAD is to be deployed, this spec is not necessary.
```

Comment:
```suggestion
<1> Optional: Specify a namespace to apply the NAD. This spec is not necessary when working in the same namespace as the deployed NAD.
```
Suggestion to avoid passive voice.

Reply:
Sort of the same thing here as above.